test/eme.test.js

```diff
@@ -797,26 +797,39 @@ QUnit.test('getLicense calls back with error for 400 and 500 status codes', func
   const getLicenseCallback = sinon.spy();
   const getLicense = defaultGetLicense({});
 
+  function toArrayBuffer(obj) {
+    const json = JSON.stringify(obj);
+    const buffer = new ArrayBuffer(json.length);
+    const bufferView = new Uint8Array(buffer);
+
+    for (let i = 0; i < json.length; i++) {
+      bufferView[i] = json.charCodeAt(i);
+    }
+    return buffer;
+  }
+
   videojs.xhr = (params, callback) => {
-    return callback(null, {statusCode: 400}, {body: 'some-body'});
+    return callback(null, {statusCode: 400}, toArrayBuffer({body: 'some-body'}));
```

Comment:
These were incorrect. As we specify that the response type should be an arraybuffer, we need to convert these to array buffer.